README.md

```diff
@@ -2091,6 +2091,27 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **[Long](https://github.com/airbnb/swift#column-width) Remove blank lines between chained functions.**  [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
+
+  <details>
+
+  ```swift
+  // WRONG
+  [0, 1, 2]
```

Comment:
Many examples in the style guide have an "outer space" theme to anchor the sample code with slightly less contrived examples. Perhaps we could come up with an example related to that theme? Something like `planets.filter { $0.isTerrestrialPlanet }.flatMap { $0.moons }` could be nice.

Reply:
It would also be nice to include something a property declaration `let innerMoons = ...`, since a bare value like this isn't valid code in most contexts.